ledger/src/blockstore_meta.rs

```diff
@@ -107,12 +107,16 @@ mod serde_compat {
     }
 }
 
+pub type Index = IndexV2;
+pub type ShredIndex = ShredIndexV2;
+pub type IndexFallback = IndexLegacy;
```

Comment:
a quick comment here might be useful. Something like "We still support falling back to reading as the legacy format in the case the new format fails"